pkg/cmd/pipelineresource/list_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelineresource
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/test"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	pipelinetest "github.com/tektoncd/pipeline/test"
+	tb "github.com/tektoncd/pipeline/test/builder"
+	tu "github.com/tektoncd/cli/pkg/test"
+)
+
+func TestPipelineResources(t *testing.T) {
+
+	pres := []*v1alpha1.PipelineResource{
+		tb.PipelineResource("test-1", "test-ns-1",
+				tb.PipelineResourceSpec("image"),
+			),
+		tb.PipelineResource("test-2", "test-ns-1",
+			tb.PipelineResourceSpec("git"),
+		),
+		tb.PipelineResource("test-3", "test-ns-1",
+			tb.PipelineResourceSpec("image"),
+		),
+		tb.PipelineResource("test-4", "test-ns-2",
+			tb.PipelineResourceSpec("image"),
+		),
+	}
+
+	tests := []struct {
+		name     string
+		command  *cobra.Command
+		args     []string
+		expected []string
+	}{
+		{
+			name: "Multiple pipeline resources",
+			command: command(pres),
+			args: []string{"list","-n","test-ns-1"},
+			expected: []string{
+				"NAME     TYPE",
+				"test-1   image",
+				"test-2   git",
+				"test-3   image",
+				"",
+			},
+		},
+		{
+			name: "Single pipeline resource",
+			command: command(pres),
+			args: []string{"list","-n","test-ns-2"},
+			expected: []string{
+				"NAME     TYPE",
+				"test-4   image",
+				"",
```

Comment:
`"", ` needed? :thinking:

Reply:
yes, because it is printing "\n" every time

Reply:
:+1: